bitnami/thanos/README.md

```diff
@@ -1343,13 +1343,17 @@ ruler:
 metrics:
   enabled: true
   serviceMonitor:
-    enabled: true
+    enabled: true 
 minio:
-  enabled: true
   auth:
-    rootUser: "admin"
-    rootPassword: "minio123"
-  defaultBuckets: "thanos"
+    rootPassword: minio123
+    rootUser: minio
+  monitoringBuckets: thanos
+  enabled: true
```

Comment:
I would keep `enabled: true` as the first element in `minio:` so you can see clearly if it is enabled or not before considering the rest of parameters.